css/style.css

```diff
@@ -1,6 +1,180 @@
+* {
+  box-sizing: border-box;
+}
+
 html,
 body {
-  font-family: montserrat-font, sans-serif;
+  font-family: 'montserrat-font', sans-serif;
   font-size: 15px;
   font-weight: normal;
+  color: #000000;
+}
+
+a {
+  text-decoration: none;
+}
+
+.btn {
+  padding: 10px 45px;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: bold;
+  text-transform: uppercase;
+  background-color: #ffffff;
```

Comment:
Codacy found an issue: [Expected "#ffffff" to be "#fff" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6093420)